mlflow/tracking/_model_registry/client.py

```diff
@@ -89,6 +89,27 @@ def list_registered_models(self,
         """
         return self.store.list_registered_models(max_results, page_token)
 
+    def search_registered_models(self,
+                                 filter_string=None,
+                                 max_results=SEARCH_REGISTERED_MODEL_MAX_RESULTS_DEFAULT,
+                                 order_by=None,
+                                 page_token=None):
+        """
+        Search for registered models in backend that satisfy the filter criteria.
+
+        :param filter_string: A filter string expression. Currently supports a single filter
```

Comment:
This doesn't seem grammatically correct -would be good to reword this (this is probably my own fault btw, since this looks copy pasted from the `store`)

Reply:
🍝

Reply:
Updated this comment. kept it very generic for client (similar to search runs).
Also generic for abstract store and more specific comment for rest and sql store implementations